### ISSUE TITLE
@org.atmosphere.config.service.Disconnect do not detect Android Chrome close Tab

### ISSUE THREAD

**Issue by umbreak** [Atmosphere/atmosphere-play#32]
Hello,

Atmosphere-play 2.1.2 with `@ManagedService`

When closing a tab with Android Chrome, the `@Disconnect` method is not fired.

`AsyncSupportListener onClose` is fired, but this is not in the context of the room, so it is not useful for me.

Try out the chat sample you provided with **Chrome browser for Android** and you will see it does not detect the closing of the tab (it does detect if in the URL bar I change the URL, but nothing else).

**Comment by umbreak** [comment]
Checking a bit further, it is detected as a **@org.atmosphere.config.service.Resume** which is obviously wrong.

**Comment by jfarcand** [comment]
@umbreak Which server are you using? That's the server that send this information to Atmosphere.

**Comment by umbreak** [comment]
It works with Firefox desktop, Firefox mobile, chrome desktop, etc. 

I am using directly Play wirhout any server un between (not apache nor lighthttp). And Play is using internally Netty

**Comment by umbreak** [comment]
Maybe could be relates wirh that issue:http://stackoverflow.com/questions/11755605/chrome-websocket-connection-not-closed-when-browser-closed ?

**Comment by umbreak** [comment]
@jfarcand , some other suggestion or I should hack the onResume to detect when it is Chrome and force the Disconnect?

**Comment by jfarcand** [comment]
@umbreak I don't have the cycle to look at it right now, but the issue seems to be related to the javascript, e.g the close message is not send by Chrome on Android. Check to see if [this code](https://goo.gl/wHXA2r) is called on Android

**Comment by umbreak** [comment]
Yes. As you suggested, the **_disconnect** method is not being called for Android Chrome when closing the browser though the android apps panel. Only in case of changing the URL on the browser, that method is called
